mParticleCore.brs

```diff
@@ -671,6 +671,14 @@ function mParticleStart(options as object, messagePort as object)
             mpBatch.ck = mparticle()._internal.storage.getCookies()
             mpBatch.das = mparticle()._internal.storage.getDas()
             mpBatch.con = mparticle()._internal.storage.getConsentState(currentMpid)
+            if (mparticle()._internal.configuration.dataPlanId <> invalid) then
+                mpBatch.ctx = {}
+                mpBatch.ctx.dpln = {}
+                mpBatch.ctx.dpln.id = mparticle()._internal.configuration.dataPlanId
```

Comment:
can you do a check to make sure they are passing in dataPlanId as a string, and if not, log an error like [so](https://github.com/mParticle/mparticle-web-sdk/blob/master/src/store.js#L206-L207)

same with dataplanversion below. mimic https://github.com/mParticle/mparticle-web-sdk/blob/master/src/store.js#L217-L219